documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,19 +51,22 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  labels:
+    createdForProviderType: <provider_type> <1>
 type: Opaque
 stringData:
-  user: <user> <1>
-  password: <password> <2>
-  cacert: | <3>
-    <engine_ca_certificate>
-  thumbprint: <vcenter_fingerprint> <4>
-EOF
+  user: <user> <2>
+  password: <password> <3>
+  cacert: | <4>
+    <ca_certificate>
+  url: <api_end_point>
+  thumbprint: <vcenter_fingerprint> <5>
```

Comment:
perhaps I don't understand the formatting, the secret format for openstack is for example:
```yaml
kind: Secret
apiVersion: v1
metadata:
  name: os-secret
  namespace: konveyor-forklift
  ownerReferences:
    - apiVersion: forklift.konveyor.io/v1beta1
      kind: Provider
      name: os-provider
      uid: ffe05f7e-7ac4-4573-962a-80d0a6ac868d
  labels:
    createdForResource: os-provider
    createdForResourceType: providers
stringData:
  username: 
  password:
  domainName:
  projectName:
  regionName:
  cacert:
type: Opaque

```

Reply:
Fixed.